app/README.md

```diff
@@ -8,7 +8,8 @@ This app will require some configuration. The API will be locked down and requir
 
 ## Super Quickstart
 
-Ensure that you have filled in all the appropriate configurations following [config/custom-environment-variables.json](config/custom-environment-variables.json) before proceeding.
+Ensure that you have filled in all the appropriate configurations shown in [config/custom-environment-variables.json](config/custom-environment-variables.json) before proceeding.
+You can configure your **database connection**, **front and back-end paths** and **authentication parameters** as well as **object storage** for file uploads with your [local environment files](config/test.json) or [openshift config maps](../openshift/README.md).
```

Comment:
Not sure if there's any value linking to the test.json as that specific file is only used in CI/CD environments. Instead, there should be a note saying create `local.json` and add in your values there or something along those lines.